app/controllers/welcome.py

```diff
@@ -0,0 +1,8 @@
+from app import app, db
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+
+class WelcomeRoutes:
+    @app.route("/")
+    def hello():
+        return "Hello World!"
```

Comment:
This will obviously need built out to return whatever FE team needs for splash page. OK to stay with Hello World for placeholder now.